packages/nuejs.org/docs/performance-optimization.md

```diff
@@ -1,9 +1,9 @@
 
 # Performance optimization
-Nue gives you performance figures of a text-only website without compromising on design. That is: "as fast as you can get".
+Nue allows you to reach performance of a text-only website without compromising on design, in other words "as fast as you can get".
 
 ## Minimalism
-Lean means fast. Develop your site with [Nue CSS best practices](css-best-practices.html) to get the cleanest and smallest codebase possible with least amount of bloat. It's quite easy to accomplish significant differences on what your codebase looks like:
+Lean, means, fast. Develop your site with [Nue CSS best practices](css-best-practices.html) to get the cleanest and smallest codebase possible with least amount of bloat. It's quite easy to achieve significant reduction in the amount of code that needs to be written:
```

Comment:
```suggestion
Lean, means fast. Develop your site with [Nue CSS best practices](css-best-practices.html) to get the cleanest and smallest codebase possible with the least amount of bloat. It's quite easy to achieve significant reduction in the amount of code that needs to be written:
```

Reply:
I was wrong. I read it too quickly as Lean, mean, fast. The second comma shouldn't be there as per your changes. And adding "the" is great too.